game/src/main/java/game/command/TankMoveCommand.java

```diff
@@ -0,0 +1,28 @@
+package game.command;
+
+import engine.action.Action;
+import engine.command.Command;
+import engine.geometry.Direction;
+import engine.geometry.Vector;
+import game.action.ChangeSpeedAction;
+import game.object.Tank;
+
+public class TankMoveCommand implements Command {
+
+    private final Tank tank;
+    private final Direction.Type direction;
+
+    private static final double speedModule = 1;
```

Comment:
static final variables are named with uppercase and words are separated with underscore

TANK_SPEED_MODULE

Please explain what is this module for?